packages/terra-form-select/wdio.conf.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 const wdioConf = require('../../node_modules/terra-toolkit/config/wdio/wdio.conf');
 
-wdioConf.config.specs = [path.join(__dirname, 'tests', 'wdio', '**', '*-spec.js')];
+wdioConf.config.specs = [path.join(__dirname, 'tests', 'wdio', '**', 'select-spec.js')];
```

Comment:
I'd revert this, this is a _frame-spec.js that wouldn't match this regex. Was the regex/matcher causing issues?

Reply:
Ohh I forgot to revert that done b5171d2